### PR TITLE
Fix theurgy stow items

### PR DESCRIPTION
**Pull request by jayRyan24**

### CODE REVIEW

File: theurgy.lic [hunk 1]
```diff
@@ -893,22 +875,22 @@ class TheurgyActions
...
          'The immaculate anloral', 'That doesn\'t appear')
     waitrt?
-    put_in_container(@water_holder)
+    DRCI.put_away_item(@water_holder, @theurgy_supply_container)
     true
   end
 
   def embarass_myself
-    return unless inside_container?('parchment')
+    return unless DRCI.inside?('parchment', @water_holder)
```
Comment (wstampley) [inline theurgy.lic:888]: Should this be 'water_holder' for your parchments?

File: theurgy.lic [hunk 2]
```diff
@@ -816,11 +798,11 @@ class TheurgyActions
     walk_to_altar_or_prayer_mat
     bput("get #{@flint_lighter}",
          'You get', 'I could not find', 'What were you referring to')
-    if inside_container?('burnt incense')
+    if DRCI.inside?('bunt incense', @water_holder)
```
Comment (wstampley) [inline theurgy.lic:801]: Should this be 'burnt' incense?
Should this be @water_holder?

File: theurgy.lic [hunk 3]
```diff
@@ -543,10 +529,10 @@ class TheurgyActions
   end
 
   def commune_kertigen
-    if inside_container?('holy oil')
-      get_from_container('holy oil')
+    if DRCI.inside?('holy oil', @water_holder)
```
Comment (wstampley) [inline theurgy.lic:532]: Again, is this @water container correct for oil?

File: theurgy.lic [hunk 4]
```diff
@@ -501,9 +487,9 @@ class TheurgyActions
     stow_hands
 
     offered_item = nil
-    if inside_container?('taffelberries')
+    if DRCI.inside?('taffelberries', @water_holder)
```
Comment (wstampley) [inline theurgy.lic:490]: @water_holder?

**Comment by rpherbig** [comment]
Good catch(es) @wstampley ! Thanks for the 👀

**Pull request merged**